src/app_service/common/network_constants.nim

```diff
@@ -271,3 +271,14 @@ var NODE_CONFIG* = %* {
   },
   "OutputMessageCSVEnabled": false
 }
+
+var STATUS_NODE_API_CONFIG* = %* {
+  "APIModules": "connector",
+  "ConnectorEnabled": true,
+  "HTTPEnabled": true,
+  "HTTPHost": "0.0.0.0",
+  "HTTPPort": 8545,
+  "WSEnabled": false,
+  "WSHost": "0.0.0.0",
+  "WSPort": 8586,
```

Comment:
How did you chose the port for the Websocket please?

Reply:
it's random, the only thing I wanted to make sure is that there is a small margin for http in case we do a port shift for a release/debug build.